### PR TITLE
dd unit test to event in common/kafka

### PR DESCRIPTION
**Pull request by mgold1234**
Signed-off-by: mgold1234 <mgold@redhat.com>

# Description

the coverage unit test for event.go is 0%.
this pr add unit test and increase the coverage to 100%
Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/common/kafka/events_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+						RedHatOrgID: common.DefaultOrgID,
+						Source:      models.SourceEdgeEventAPI,
+						SpecVersion: "v1",
+						Subject:     image.Name,
+						Time:        time.Now().Format(time.RFC3339),
+						Type:        models.EventTypeEdgeImageRequested,
+					}
+					event := kafkacommon.CreateEdgeEvent(common.DefaultOrgID, models.SourceEdgeEventAPI, image.RequestID, models.EventTypeEdgeImageRequested, image.Name, edgePayload)
+					Expect(event.Data).To(Equal(cloudEvent.Data))
+					Expect(event.RedHatOrgID).To(Equal(cloudEvent.RedHatOrgID))
```
Comment (ldjebran) [inline pkg/common/kafka/events_test.go:75]: maybe like this: 
```suggestion
					Expect(event.Data).To(Equal(edgePayload))
					Expect(event.RedHatOrgID).To(Equal(common.DefaultOrgID))
```

and remove cloudEvent variable, 

what do you think ?

**Review by ldjebran — APPROVED** [review]
lgtm

File: pkg/common/kafka/events_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		config.Init()
+
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+	Describe("Get Consumer", func() {
```
Comment (tomershinhar) [inline pkg/common/kafka/events_test.go:30]: i am not sure i understand the naming here, this test is about creating an event, not getting a consumer so "Get consumer" is not correct.

also "One does not exist yet" seems to be unrelated to the test.
as a final note, i believe there is no need fo both `When` and `Context` here, you can drop `Context`

Comment (de1987) [reply pkg/common/kafka/events_test.go:30]: adding more
if your goal is to test just function I would suggest using something like this
`#CreateEdgeEvent`

**Review by tomershinhar — APPROVED** [review]
LGTM

**Review by ldjebran — APPROVED** [review]

File: pkg/common/kafka/events_test.go (hunk 2 above)
Comment (ldjebran) [reply pkg/common/kafka/events_test.go]: I believe that this is resolved
